NEWS.md

```diff
@@ -1,3 +1,18 @@
+# scoringutils 1.1.5
+
+## Feature updates
+- Added a new function, `set_forecast_unit()` that allows the user to set the 
+forecast unit manually. The function removes all columns that are not relevant
+for uniquely identifying a single forecast. If not done manually, `scoringutils` 
+attempts to determine the unit
+of a single automatically by simply assuming that all column names are
+relevant to determine the forecast unit. This can lead to unexpected
+behaviour, so setting the forecast unit explicitly can help make the code
+easier to debug and easier to read (see issue #268). 
+When used as part of a workflow,
+`set_forecast_unit()` can then directly be piped into `check_forecasts()` to
```

Comment:
can be vs can then

Reply:
removed "then" assuming that's what you meant